src/lib.rs

```diff
@@ -6,17 +6,18 @@ extern crate serde_derive;
 extern crate serde_json;
 extern crate toml;
 
-pub mod build;
 pub mod bindgen;
-pub mod readme;
+pub mod build;
+pub mod emoji;
 pub mod manifest;
+pub mod npm;
 pub mod progressbar;
-pub mod emoji;
+pub mod readme;
 
 use std::fs;
 
-use failure::Error;
 use console::style;
+use failure::Error;
```

Comment:
my editor just started reordering these when i installed the rust ide.. 🤷‍♀️

Reply:
it's super fine <3

Reply:
I like having those fields alphabetical personally so this seems great to me :D